components/script/dom/gpudevice.rs

```diff
@@ -769,6 +771,7 @@ impl GPUDeviceMethods for GPUDevice {
             Some(wgpu_pipe::RenderPipelineDescriptor {
                 label: convert_label(&descriptor.parent.parent),
                 layout,
+                cache: None,
```

Comment:
So the idea is this will move to the backend thread as well right?

Reply:
Yes, when wgpu changes how renderpasses work.